features/AssetController.feature

```diff
@@ -80,10 +80,10 @@ Feature: DeviceManager asset controller
       | size     | 3                          |
     Then I should receive a "measures" array of objects matching:
     # there is 6 measures with the 3 from fixtures
-      | _source.origin.assetId    | _source.origin.id                        |
-      | "container-FRIDGE-linked" | "DummyMultiTemp-attached_ayse_linked_1"  |
-      | "container-FRIDGE-linked" | "DummyMultiTemp-attached_ayse_linked_1"  |
-      | "container-FRIDGE-linked" | "DummyMultiTemp-attached_ayse_linked_1"  |
+      | _source.asset._id         | _source.origin.id                       |
```

Comment:
You should also check that the property of the `_source` object are also present

Reply:
done